starport/cmd/chain.go

```diff
@@ -0,0 +1,20 @@
+package starportcmd
+
+import "github.com/spf13/cobra"
+
+// NewChain returns a command that groups sub commands related to compiling, serving
+// blockchains and so on.
+func NewChain() *cobra.Command {
+	c := &cobra.Command{
+		Use:     "chain [command]",
+		Short:   "Compile, serve blockchains and so on",
```

Comment:
> Short: "Compile, serve blockchains **and so on**",

Not a big fan of this wording, `Compile, serve blockchains and interact with running blockchain`? (Thinking that faucet is about this and maybe future eventual command as well)

Reply:
I agree, I couldn't find the best wording for the second part of the phease but `interact with running blockchain` seems confusing to me.

Reply:
"Build, initialize, and start a blockchain"

We don't have to mention everything that's in the namespace, just the most important things.

Reply:
```suggestion
		Short:   "Build, initialize, and start a blockchain",
```